Product/DBScripts/nhincdb/nhincdb.sql

```diff
@@ -562,6 +562,9 @@ VALUES
 ("patientDiscovery.xhtml", "Cross-Query Gateway Client", 0, 1),
 ("patientDiscovery.xhtml", "Cross-Query Gateway Client", 0, 2),
 ("patientDiscovery.xhtml", "Cross-Query Gateway Client", 0, 3),
+("loadTestData.xhtml", "Load Test Data", 0, 1),
+("loadTestData.xhtml", "Load Test Data", 0, 2),

```

Comment:
Need to add these in nhincdb_oracle.sql as well

Reply:
done